drivers/lora_lbm/Kconfig

```
@@ -55,7 +55,26 @@ config LORA_BASICS_MODEM_DRIVERS_EVENT_TRIGGER_DIRECT
 	depends on GPIO
 	select LORA_BASICS_MODEM_DRIVERS_EVENT_TRIGGER
 
-endchoice
+endchoice # LORA_BASICS_MODEM_DRIVERS_EVENT_TRIGGER
+
+choice
+	prompt "Event trigger type"
+	default LORA_BASICS_MODEM_DRIVERS_EVENT_TRIGGER_TYPE_EDGE_TO
+	help
+	  Specify the type of triggering to be used by the LORA_BASICS_MODEM_DRIVERS driver - edge to or level based. There is difference in power consumption of ~20 uA between them.
+
+config LORA_BASICS_MODEM_DRIVERS_EVENT_TRIGGER_TYPE_NONE
+	bool "No trigger type"
+
+config LORA_BASICS_MODEM_DRIVERS_EVENT_TRIGGER_TYPE_EDGE_TO
+	bool "Edge to active / inactive trigger"
+	depends on LORA_BASICS_MODEM_DRIVERS_EVENT_TRIGGER
+
```

```suggestion
config LORA_BASICS_MODEM_DRIVERS_EVENT_TRIGGER_TYPE_EDGE
	bool "Edge to active / inactive trigger"
	depends on LORA_BASICS_MODEM_DRIVERS_EVENT_TRIGGER

```
- `..._TYPE_EDGE_TO` is IMHO not okay. I would change it to `..._TYPE_EDGE`
- Update help text if you'll add this change.

---

You can have the trigger defined as `GPIO_INT_EDGE_TO_ACTIVE` or as `GPIO_INT_EDGE_RISING`. Naming type as `LORA_BASICS_MODEM_DRIVERS_EVENT_TRIGGER_TYPE_EDGE` does not distinguish between the two, therefore, I added `_TO`. If you have any other suggestion, feel free to add - I did not think of a better option.